main.go

```diff
@@ -7,6 +7,8 @@ import (
 )
 
 func main() {
+	log.SetFlags(0)
```

Comment:
Can you add the body of this PR as an inline comment? It would help readers understand why we're doing that.